src/SymbolPaths/include/SymbolPaths/QSettingsWrapper.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOL_PATHS_QSETTINGS_WRAPPER_H_
+#define SYMBOL_PATHS_QSETTINGS_WRAPPER_H_
+
+#include <filesystem>
+#include <vector>
+
+namespace orbit_symbol_paths {
+
+void SavePaths(const std::vector<std::filesystem::path>& paths);
```

Comment:
maybe add a short comment mentioning usage of QT settings and thread-safety?

Reply:
done